deploy/scripts/src/utils/functions/functions-mobile-journey.ts

```diff
@@ -126,14 +126,16 @@ function getBackendUrl (path: string, query?: Record<string, string>): string {
 }
 
 export function startDcmawJourney (): void {
-  const res = http.get(getFrontendUrl('/selectDevice'))
+  const res = http.get(getFrontendUrl('/selectDevice'), {
+    tags: { name: 'Start DCMAW Journey' }
+  })
   isStatusCode200(res)
   isPageContentCorrect(res, 'Are you on a computer or a tablet right now?')
   isPageRedirectCorrect(res, '/selectDevice')
 }
 
 export function checkSelectDeviceRedirect (device: DeviceType): void {
-  const res = http.post(getFrontendUrl('/selectDevice'), { 'select-device-choice': device })
+  const res = http.post(getFrontendUrl('/selectDevice'), { 'select-device-choice': device }, { tags: { name: 'Select device: ' + device } })
```

Comment:
```suggestion
  const res = http.post(getFrontendUrl('/selectDevice'), { 'select-device-choice': device }, { tags: { name: '/selectDevice'} })
```
Would suggest removing variables from all tags - we want to group these requests together regardless of what option we are sending in the POST